keygen/src/keygen.rs

```diff
@@ -349,7 +388,47 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                         .validator(is_parsable::<usize>)
                         .default_value(&default_num_threads)
                         .help("Specify the number of grind threads"),
-                ),
+                )
+                .arg(
+                    Arg::with_name("mnemonic")
+                        .long("mnemonic")
+                        .alias("use-mnemonic")
+                        .help("Generate using a mnemonic key phrase"),
+                )
+                .arg(
+                    Arg::with_name("word_count")
+                        .requires_all(&["mnemonic"])
```

Comment:
This requirement breaks using grind *without* a mnemonic because args with `default_value`s are always present. Just remove this, I think.
Ditto for "language"

Reply:
I didn't realize that args with default values are always considered to be present, thank you for noting that.

The reason for including word_count and language is to allow the same options that are available for generating a new keypair to be available when grinding. 

This goes back to my belief that 'new' and 'grind' really should logically be the same thing, with the mechanism being different but the end result essentially being the same, the only difference being whether the user cares about the prefix, suffix, or prefix-and-suffix of the generated public key ...

Reply:
Sorry, again I was only talking about the single line: `.requires_all(&["mnemonic"])`
I'll be more specific in future reviews!

Reply:
No problem, I am so new to Rust and this code base that I think I am easily confused that this point.  Thank you for your patience.